packages/documentation/.storybook/preview.js

```diff
@@ -51,28 +61,17 @@ export const parameters = {
       excludeDecorators: true,
     },
     transformSource(snippet) {
-      return (
-        snippet
-          // remove react fragments
-          .replace(/(<>|<\/>)/g, '')
-
-          // remove "key" attributes
-          .replace(/(\t+|\s+)?key=".*"/g, '')
-
-          // remove "{' '}" placeholders
-          .replace(/{' '}/g, ' ')
-
-          // repalce noRefCheck functions
-          .replace(/function noRefCheck\(\)\s?{}/g, '() => {}')
-
-          // remove brackets from "{value}" attribute-values
-          .replace(/([a-zA-Z][a-zA-Z0-9-_:.]+)={([^}]*}?)}/g, (_m, g1, g2) => `${g1}="${g2}"`)
-
-          // replace "className" attributes with "class"
-          .replace(/className/g, 'class')
+      const reactElement = <JsxParser jsx={snippet} renderInWrapper={false}/>;
+      const htmlString = renderToString(reactElement);
 
-          // replace "htmlFor" attributes with "for"
-          .replace(/htmlFor/g, 'for')
+      return beautify.html(
```

Comment:
Can we use pretter here? We use it allready in some other files and it would be nice to not have multiple packages installed, that literally do the same thing.
If the output is totally different, I would off course accept the beautify package ;)

Reply:
I used your implementation except for the `.replace()` methods which no longer seem useful since `renderToStaticMarkup` already handles these changes.